src/pages/FriendsActivitiesPage.js

```diff
@@ -30,10 +44,28 @@ export default function FriendsActivitiesPage({
         </ArrowBackButton>
       </Header>
       <Main>
+        {categoryTagsAndAll.length > 2 ? (
+          <ScrollCategories title="filter options">
+            {categoryTagsAndAll.map((category, index) => {
+              return (
+                <CategoryButton
+                  key={index}
+                  onClick={() => onFilter(category)}
+                  active={category === currentFilter}
+                >
+                  {category}
+                </CategoryButton>
+              );
+            })}
+          </ScrollCategories>
+        ) : (
+          ''
+        )}
```

Comment:
since you dont render anything else here, you can use the && operator in line 47